### PR TITLE
Feature/update image cards unit tests

### PR DESCRIPTION
**Pull request by Masonfrith**
## Description

Change ImageCards tests to use the store.
Add a few more tests to test with non-default data, and with empty array
## Motivation and Context

Updating tests to test more and to test they work with the store
## How Has This Been Tested?

- [x] Unit test
- [ ] NightWatch test
- [ ] Coverage testing
- [ ] Manual testing

## Test Cases and test paths

these are unit tests, run from file imageCards.spec.js
## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/store/modules/slide/index.js [hunk 1]
```diff
@@ -252,18 +252,6 @@ const mutations = {
...
-  // takes a template object as payload. set template of currentslide, and dirty it.
-  [CURRENT_SLIDE.SET_TEMPLATE] (state, payload) {
-    state.currentSlide.template = payload
-    state.isCurrentSlideDirty = true
-  },
-
-  // takes timeout as payload (int), and sets timeout for currentslide, dirty it.
-  [CURRENT_SLIDE.SET_TIMEOUT] (state, payload) {
-    state.currentSlide.timeout = payload
-    state.isCurrentSlideDirty = true
```
Comment (peggyopal) [inline src/store/modules/slide/index.js]: Are these supposed to be removed?

File: test/unit/specs/ImageCards.spec.js [hunk 2]
```diff
@@ -1,25 +1,106 @@
...
+  before(function () {
+    Vue.use(Vuex)
+    Constructor = Vue.extend(ImageCards)
+    const store = new Vuex.Store(slide)
+    vm = new Constructor({ store }).$mount()
     sinon.spy(vm, '$emit')
+    mockCurrentSlide = {
+      images: [
+        {
+          src: 'https://picsum.photos/200/300/?image=92'
```
Comment (peggyopal) [inline test/unit/specs/ImageCards.spec.js]: Are these links ever going to expire?

File: src/store/modules/slide/index.js (hunk 1 above)
Comment (KingMackle) [reply src/store/modules/slide/index.js]: Yes these were missed by a merge, they shouldn't exist in the mutations according to the new slide object schema.

File: test/unit/specs/ImageCards.spec.js (hunk 2 above)
Comment (KingMackle) [reply test/unit/specs/ImageCards.spec.js]: We hope not, at least for the next two months https://picsum.photos/

File: src/store/modules/slide/index.js (hunk 1 above)
Comment (Masonfrith) [reply src/store/modules/slide/index.js]: Should thus be removed from the pr then?

Comment (KingMackle) [reply src/store/modules/slide/index.js]: You can keep them there, because master has them, and that is not correct. So you are right to include the removal of these two mutations as part of this PR
